### DISCUSSION TITLE
The string attribute must have a value during initialization, otherwise content size is incorrect

### DISCUSSION THREAD

**Discussion by Mx-Iris** [krzyzanowskim/STTextView#30] [Questions]
https://github.com/krzyzanowskim/STTextView/assets/61279231/f602da1a-54dd-4f39-9656-4bbc73368d8e

**Comment by krzyzanowskim** [comment]
thank you!

> otherwise content size is incorrect

I understand from the video it's about the height?

**Reply by Mx-Iris** [reply]
Yes, not assigning a string attribute initially results in incorrect height and incomplete content display

https://github.com/krzyzanowskim/STTextView/assets/61279231/cf064a09-8ad9-453a-8c3e-b64c252754c2

**Reply by krzyzanowskim** [reply]
I see. I'll take a look into it. For now I suggest set `heightTracksTextView = true`

**Reply by JohJakob** [reply]
I’m having a very similar issue in SwiftUI (using `TextView`). Is there a similar workaround, e.g., a way to set `heightTracksTextView` from SwiftUI land?

**Reply by krzyzanowskim** [reply]
>  a way to set heightTracksTextView from SwiftUI land

that is not exposed. I have nothing against it. If you want to expose it in the SwiftUI interface I'm happy to merge the PR, or you can wait few days when I back to it

**Reply by JohJakob** [reply]
Your recent changes have resolved at least my issues in SwiftUI. The content size is now correct 🙏🏻

**Comment by Mx-Iris** [comment]
The issue has been solved
